packages/components/src/components/badge/badge.stories.ts

```diff
@@ -26,110 +37,56 @@ export default {
  */
 
 export const Default = {
+  name: 'Default',
   render: (args: any) => {
     return generateTemplate({ args });
   }
 };
 
 /**
- * The badge in all possible combinations of `variant` and `size`.
- */
-
-export const VariantAndSize = {
-  name: 'Variant × Size',
-  parameters: { controls: { exclude: ['variant', 'size'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'variant' },
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
-};
-
-/**
- * The badge in all possible combinations of `variant` and `inverted`.
+ * Use the `size` attribute to set the badge’s size.
```

Comment:
How about we keep the same order?

First Variant then below Size? Like we have it for sd-button?

Reply:
Updated